worlds/oot_soh/location_access/overworld/death_mountain_crater.py

```diff
@@ -6,56 +6,76 @@
 if TYPE_CHECKING:
     from worlds.oot_soh import SohWorld
 
+
 class EventLocations(str, Enum):
     DMC_GOSSIP_STONE_FAIRY = "Death Mountain Crater Gossip Stone Fairy"
     DMC_BEAN_PLANT_FAIRY = "Death Mountain Crater Bean Plant Fairy"
 
 
 def set_region_rules(world: "SohWorld") -> None:
-    player = world.player
-    
-    ## Death Mountain Crater Upper Nearby
+
+    # Death Mountain Crater Upper Nearby
```

Comment:
We've had the double ##'s on every region file to separate it from the other comments there. Is this really neccesary to remove?

Reply:
Probably not 100% necessary, I can add them back if you want

Reply:
Yeah I think I'd prefer that